app/src/main/java/org/schabi/newpipe/database/playlist/model/PlaylistRemoteEntity.java

```diff
@@ -72,10 +74,18 @@ public PlaylistRemoteEntity(final PlaylistInfo info) {
 
     @Ignore
     public boolean isIdenticalTo(final PlaylistInfo info) {
-        return getServiceId() == info.getServiceId() && getName().equals(info.getName()) &&
+        boolean returnMe;
```

Comment:
The variable `returnMe` is probably useless as it can be replaced by a simple `return` everywhere it is assigned (lines 79 and 84).

Reply:
true, it was kinda a sloppy fix, and to be honest uploader will never change so It can just be removed